db/migrations/schema-objects/20220411101455.do.add-12x12-to-trial-dev-form.sql

```diff
@@ -0,0 +1,32 @@
+UPDATE mtc_admin.checkForm
+SET formData = '[
```

Comment:
the date in the file name for this migration is nearly a month old - is this correct?

Reply:
Yes